Source/Widgets/Cesium3DTilesInspector/Cesium3DTilesInspectorViewModel.js

```diff
@@ -325,11 +325,18 @@ define([
                 if (value) {
                     that._eventHandler.setInputAction(function(e) {
                         var picked = scene.pick(e.endPosition);
-                        if (picked instanceof Cesium3DTileFeature) {
+                        if (picked && defined(picked.content)) {
                             that.feature = picked;
+                            var position;
+                            if (scene.pickPositionSupported) {
+                                position = scene.pickPosition(e.endPosition);
+                                that._tileset._debugPickPosition = position;
```

Comment:
Yeah I wasn't sure about this either or how strict we get for tightly-coupled areas like this. Maybe they should public but still marked as private for exclusion from doc?